packages/ethereum-contracts/scripts/deploy-framework.js

```diff
@@ -436,6 +438,18 @@ module.exports = eval(`(${S.toString()})({skipArgv: true})`)(async function (
         }
     }
 
+    // deploy CFAv1Forwarder
+    await deployAndRegisterContractIf(
+        CFAv1Forwarder,
+        `CFAv1Forwarder.${protocolReleaseVersion}`,
```

Comment:
similar to loader:

```
    await deployAndRegisterContractIf(
        SuperfluidLoader,
        "SuperfluidLoader-v1",
```

The forwarder should be for all protocol release versions.